src/uneditableTextField/UneditableTextField.tsx

```diff
@@ -10,13 +10,15 @@ export type UneditableTextFieldProps = Readonly<{
   addOn?: JSX.Element
   href?: JSX.Element | string
   ellipsis?: boolean
+  isPlaceholder?: boolean
```

Comment:
I _strongly_ suggest that you use `placeholder` native attribute from HTML instead of creating a prop.
By passing a placeholder prop, it can be styled with CSS
See https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder

Reply:
Unfortunately, the generated HTML is a `div`, so `::placeholder` is not applicable in this case 😬 
But as discussed with Vince, this component should be burnt, let's tackle that during Chapter!
🔥 
It could simply be a button with a special skin 🤷

Reply:
+1000 on removing this horrible component one day, or renaming it to something like FakeTextFieldButton (it's a button, not a textfield)

Reply:
😱